pkg/controller/persistentvolume/persistentvolume_index.go

```diff
@@ -117,16 +92,20 @@ func (pvIndex *persistentVolumeOrderedIndex) findByClaim(claim *api.PersistentVo
 				continue
 			}
 
-			if claim.Name == volume.Spec.ClaimRef.Name && claim.Namespace == volume.Spec.ClaimRef.Namespace && claim.UID == volume.Spec.ClaimRef.UID {
-				// exact match! No search required.
+			if isVolumeBoundToClaim(volume, claim) {
```

Comment:
> Don't see how to add this comment to persistentvolume_controller.go where it belongs since that file in this pr doesn't contain the func isVolumeBoundToClaim

That's odd: `not shown because the diff is too large. Please use a local Git client to view these changes.`
@pmorie, how did you review the file?